HW6/Lecture6.cpp

```diff
@@ -0,0 +1,141 @@
+﻿
+#include <iostream>
+#include <cctype>
+using namespace std;
+int main()
+{
+	//Task 1
+	cout << "Task I" << endl;
+	int a1;
+	int d;
+	int an;
+	cout << "Enter initial number: ";
+	cin >> a1;
+	cout << "Enter step of progression: ";
+	cin >> d;
+	cout << "Enter last number: ";
+	cin >> an;
+	for (int i = a1; i <= an; i = i + d)
+	{
+		cout << i << endl;
+	}
+
+	//Task 2
+	cout << "Task II" << endl;
+	int a = 0;
+	int b = 1;
+	int c = 0;
+	int n = 0;
+	cout << "Enter number: ";
+	cin >> n;
+	while (a < n)
+	{
+		a = b;
+		b = c;
+		c = a + b;
+		cout << c << " ";
+		a++;
+
+	}
```

Comment:
It doesn't work well. You should check your algorithm when n=20. You should get 20 numbers, but you have only 10